### PR TITLE
Merge dev into MSAL release/0.5.0 branch

### PR DESCRIPTION
**Pull request by oldalton**

### CODE REVIEW

File: common_version.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+dev
```
Comment (jasoncoolmax) [inline common_version.txt]: what is this file used for? :) should it be dev?

File: changelog.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Version 1.0.11
+------------
+* Apply hotfix 2.7.9 for Mac OS to query WPJ cert using issuers from authentication challenge
```
Comment (jasoncoolmax) [inline changelog.txt]: are we going to maintain the change log here? or we just want to rely on the change logs in the Gitnub Release page?

Comment (oldalton) [reply changelog.txt]: yes. The changeling here helps us find which version of common core particular ADAL or MSAL shipped with. Currently you need to check the hash and lookup the hash, which is time consuming. Updated the file with previous/historic changelog.

File: common_version.txt (hunk 1 above)
Comment (oldalton) [reply common_version.txt]: Removed, it got replaced with changelog.

**Pull request merged**